src/partials/_animal--owl.hbs

```diff
@@ -0,0 +1,120 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="-120 0 1000 1800" width="500px" class="c-owl">
+  <style>
+    .owl-1{fill:#F7BC60;} .owl-2{fill:#FFFFFF;} .owl-3{fill:#CDBBA2;} .owl-4{fill:#80561B;} .owl-5{fill:#9A8B7D;} .owl-6{fill:#726658;} .owl-7{fill:#4B3900;} .owl-8{fill:#3C2415;} .owl-9{fill:#D9CFC0;} .owl-10{fill:#FFCF82;} .owl-11{fill:#FDBA63;}
```

Comment:
Since this is an inline SVG, this CSS could be a part of your Sass files if you’d like.